upcloud/testdata/upcloud_kubernetes/kubernetes_labels_s2.tf

```diff
@@ -29,6 +29,7 @@ resource "upcloud_kubernetes_cluster" "main" {
   name                    = "${var.basename}cluster"
   network                 = upcloud_network.main.id
   zone                    = var.zone
+  storage_encryption      = "data-at-rest"
```

Comment:
Step 1 sets `storage_encryption`, so set it also here and in step 3 to avoid replacing the cluster during the test case.